test/powershell/Modules/Microsoft.PowerShell.Security/AmsiInterface.Tests.ps1

```diff
@@ -0,0 +1,42 @@
+﻿# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+try
+{
+    $defaultParamValues = $PSDefaultParameterValues.Clone()
+    $PSDefaultParameterValues["it:Skip"] = !$IsWindows
+
+    Describe "AMSI scan should detect suspicious content" -Tags 'Feature','RequireAdminOnWindows' {
+
+        BeforeAll {
+            [System.Management.Automation.Internal.InternalTestHooks]::SetTestHook("UseDebugAmsiImplementation", $true)
+        }
+
+        AfterAll {
+            [System.Management.Automation.Internal.InternalTestHooks]::SetTestHook("UseDebugAmsiImplementation", $false)
+        }
+
+        It "Verifies AMSI scan detects debug suspicious content" {
+
+            $EICAR_STRING_B64 = "awZ8EmMWc3JjaAdvY2lrBgcbY20aBHBwGgROF3Z6cHJhHmBncn13cmF3HnJ9Z3plemFmYB5ndmBnHnV6f3YSF3sYexk= "
+            $bytes = [System.Convert]::FromBase64String($EICAR_STRING_B64)
+            $EICAR_STRING = -join ($bytes | % { [char]($_ -bxor 0x33) })
+            try
+            {
+                Invoke-Expression -Command "echo '$EICAR_STRING'"
+                throw "No exception thrown!"
+            }
+            catch
+            {
+                $_.FullyQualifiedErrorId | Should Be "ScriptContainedMaliciousContent,Microsoft.PowerShell.Commands.InvokeExpressionCommand"
```

Comment:
Why we don't use `ShouldBeErrorId`?

Reply:
The tests were already like this.  I feel changing all tests is outside the scope of this change.  Please create a new issue.

Reply:
Can one of you file an issue to fix the test?

Reply:
Done #6151